specs/linux/md_devices.table

```diff
@@ -29,9 +29,9 @@ schema([
     Column("reshape_progress", TEXT, "Progress of the reshape activity"),
     Column("reshape_finish", TEXT, "Estimated duration of reshape activity"),
     Column("reshape_speed", TEXT, "Speed of reshape activity"),
-    Column("check_array_progress", TEXT, "Progress of the resync activity"),
-    Column("check_array_finish", TEXT, "Estimated duration of resync activity"),
-    Column("check_array_speed", TEXT, "Speed of resync activity"),
+    Column("check_array_progress", TEXT, "Checking for the array activity in progress"),
```

Comment:
I think these should read:

"Progress of the check array activity"
"Estimated duration of the check array activity"
"Speed of the check array activity"